hack/change-stored-version.sh

```diff
@@ -0,0 +1,176 @@
+#!/usr/bin/env bash
```

Comment:
This was something I hacked together to help incrementally update all the v1alpha2 references.
I should probably remove this from the PR once it's been approved.

Reply:
would be nice to have this somewhere for v2 ;)

Reply:
I'll move it to another branch.